features/resending_credit_memo.feature

```diff
@@ -0,0 +1,26 @@
+@refunds
+Feature: Resending credit memo
+    In order to provide a generated credit memo to the customer
+    As an Administrator
+    I want to be able to send credit memo more that once
+
+    Background:
+        Given the store operates on a single green channel in "United States"
+        And the store has a product "Mr. Meeseeks T-Shirt" priced at "$10.00"
+        And the store has a product "Portal Gun" priced at "$20.00"
+        And the store allows shipping with "Galaxy Post"
+        And the store allows paying with "Space money"
+        And there is a customer "rick.sanchez@wubba-lubba-dub-dub.com" that placed an order "#00000022"
+        And the customer bought a single "Mr. Meeseeks T-Shirt"
+        And the customer bought a single "Portal Gun"
+        And the customer chose "Galaxy Post" shipping method to "United States" with "Space money" payment
+        And the order "#00000022" is already paid
+        And 1st "Mr. Meeseeks T-Shirt" product from order "#00000022" has already been refunded with "Space money" payment
+        And 1st "Portal Gun" product from order "#00000022" has already been refunded with "Space money" payment
+        And I am logged in as an administrator
+
+    @ui @email
+    Scenario: Resend credit memo to a customer
+        When I browse credit memos
+        And I resend credit memo from order "#00000022"
+        Then the 3 emails to "rick.sanchez@wubba-lubba-dub-dub.com" with credit memo should be sent
```

Comment:
In this scenario, I would be for checking only the email that was sent again, not all of emails (it is not the scope of this scenario)
-> `Then the email with credit memo should be sent to "rick.sanchez@wubba-lubba-dub-dub.com"`

Reply:
okay, but I have no idea how to check only the email which was sent with "resend button".
Some suggestion :)?

Reply:
To be honest, I don't have any good idea. Checking only the last email, won't be a proper test. Maybe you could remove spooled messages before clicking the `Resend` button, but it is probably not the best idea. 

Any idea @Zales0123 @bartoszpietrzak1994 ?